client/settings/transactions/index.js

```diff
@@ -170,6 +173,7 @@ const Transactions = ( { setTransactionInputsValid } ) => {
 									data-testid={
 										'store-name-bank-statement-kana'
 									}
+									__nextHasNoMarginBottom
```

Comment:
I missed some TextControl components:
`36px default size for wp.components.TextControl is deprecated since version 6.8 and will be removed in version 7.1. Note: Set the `__next40pxDefaultSize` prop to true to start opting into the new default size, which will become the default in a future version.`
```suggestion
									__nextHasNoMarginBottom
									__next40pxDefaultSize
```